src/components/ColorPicker/ColorPicker.vue

```diff
@@ -32,10 +32,81 @@ actual pickers:
 
 ### Usage
 
+* Using v-model and passing in an HTML element that will be treated as a trigger:
+
+```vue
+<template>
+	<ColorPicker v-model="color">
+		<button> Click Me </button>
+	<ColorPicker />
+</template>
+<script>
+export default {
+	data() {
+		return {
+			color: '#0082c9'
+		}
+	},
+}
+</script>
+```
+
+* Using `:value.sync` and binding to the open state to the open prop:
+(Note: if you see an empty example box it's because you've previously clicked somewhere in
+the page and closed the popover. Just add a whitespace to the code in the sandbox to 
+refresh the output)
+
+```vue
+<template>
+	<ColorPicker :value.sync="color" :open="open" />
+</template>
+<script>
+export default {
+	data() {
+		return {
+			color: '#0082c9'
+		}
+	},
+	props: {
+		open: {
+			type: Boolean,
+			default: true
+		}
+	}
+}
+</script>
 ```
-<ColorPicker v-model="color" />
-<ColorPicker :value.sync="color" />
-<ColorPicker :value="color" @change="updateColor" />
+
+* Using v-bind for both color and open state and emitting an event that updates the color
+(Note: if you see an empty example box it's because you've previously clicked somewhere in
+the page and closed the popover. Just add a whitespace to the code in the sandbox to 
+refresh the output)
+
+```vue
+<template>
+	<ColorPicker :value="color" @change="updateColor" :open="open" />
+</template>
+<script>
+export default {
+	data() {
+		return {
+			color: '#0082c9'
+		}
+	},
+	props: {
+		open: {
+			type: Boolean,
+			default: true
+		}
+	},
+	methods: {
+		updateColor(e) {
+			this.color = e.color
```

Comment:
@change is bound to `this.$emit('change', this.currentColor)`
so it should be `this.color = e`, no?